source/ui/datasources/ui_blockedplayers_datasource.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+#include <Rocket/Controls/DataSource.h>
+#include <cstdint>
+#include <vector>
+
+namespace WSWUI
+{
+
+	typedef struct {
+		char steamid[18];
```

Comment:
We are no longer in c land, so you can use some less horrible stuff than fixed size buffers here.